environment/templates/chart_apps.yaml

```diff
@@ -85,6 +94,21 @@ spec:
       valueFiles:
         - {{ printf "%s/%s/%s/values.yaml" $.Values.repository.envs_from_charts $.Values.env.name $chart_app_name }}
     {{- end }}
+    {{- if $chart_app_obj.parameters }}
+    {{- with $chart_app_obj.parameters }}
+      parameters:
+    {{- tpl (toYaml .) $ | nindent 8 }}
+    {{- end }}
+    {{- end }}
+    {{- if $chart_app_obj.valuesObject }}
+    {{- with $chart_app_obj.valuesObject }}
```

Comment:
```
    {{- if $chart_app_obj.valuesObject }}  = if true?
    {{- with $chart_app_obj.valuesObject }} = if exists?
```
in the sense of both lines doing the same thing, why do we need both lines?

Reply:
hmm yeah let me try

Reply:
fixed please check @devops-dmitry